test/suite/test_verbose04.py

```diff
@@ -0,0 +1,169 @@
+        # WT_VERBOSE_DEBUG_1 (1). Ensuring the verbose output generated matches any of the existing verbose categories.
+        # 'all' category.
+        with self.expect_verbose(['all:1'], self.all_verbose_categories, self.is_json) as conn:
+            # Perform a set of simple operations to generate verbose messages from different categories.
+            uri = 'table:test_verbose04_all'
+            session = conn.open_session()
+            session.create(uri, self.collection_cfg)
+            c = session.open_cursor(uri)
+            c['all'] = 'all'
+            c.close()
+            session.create(uri, self.collection_cfg)
+            session.compact(uri)
+            session.close()
+            
+        # At this time, no verbose messages should be generated with the following set of operations and the verbosity level 
+        # WT_VERBOSE_INFO (0), hence we don't expect any output.
+        with self.expect_verbose(['all:0'], self.all_verbose_categories, self.is_json, False) as conn:
+            uri = 'table:test_verbose04_all'
+            session = conn.open_session()
+            session.create(uri, self.collection_cfg)
+            c = session.open_cursor(uri)
+            c['all'] = 'all'
+            c.close()
+            session.close()
+            
+        # Test passing another single verbose category, 'all' with different verbosity levels.
+        # Since there are verbose messages with the category WT_VERB_COMPACT and the verbosity
+        # levels WT_VERBOSE_INFO (0) through WT_VERBOSE_DEBUG_5 (5), we can test them all.
+        cfgs = ['all:0', 'all:1', 'all:2', 'all:3', 'all:4', 'all:5']
+        for cfg in cfgs:
+            with self.expect_verbose([cfg], self.all_verbose_categories, self.is_json) as conn:
+                # Create a simple table to invoke compaction on. We aren't doing anything
+                # interesting with the table, we want to simply invoke a compaction pass to generate
+                # verbose messages.
+                uri = 'table:test_verbose04_all'
+                session = conn.open_session()
+                session.create(uri, self.collection_cfg)
+                session.compact(uri)
+                session.close()
+                
+    # Test use cases passing multiple verbose categories, ensuring we only produce verbose output
+    # for specified categories.
+    def test_verbose_multiple(self):
+        self.close_conn()
+        # Test passing multiple verbose categories, being 'api' & 'all' & 'version' with different dedicated
+        # verbosity levels to each category. Ensuring the only verbose output generated is related
+        # to those two categories.
+        cfgs = ['api:0,all:1,version:0', 'version:0,all,api:0']
+        
+        #all_verbose_categories_except_api_and_version contains all verb flag(except WT_VERB_API & WT_VERB_VERSION)
+        all_verbose_categories_except_api_and_version = self.all_verbose_categories
```

Comment:
This creates a reference to `self.all_verbose_categories`, so the 
` all_verbose_categories_except_api_and_version.remove('WT_VERB_API')`
line below also removes `WT_VERB_API` from `self.all_verbose_categories` and `WT_VERB_API` it will be missing from the array in following tests.

We can fix this with:
```suggestion
        all_verbose_categories_except_api_and_version = self.all_verbose_categories.copy()
```